wavfile test flaky

### What happened?

after running a couple dozens time:

```
    Test #112: qa_wavfile .......................***Failed    0.13 sec
..F..ss...
======================================================================
FAIL: test_003_checkwav_append_copy (__main__.test_wavefile.test_003_checkwav_append_copy)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/root/gnuradio/gr-blocks/python/blocks/qa_wavfile.py", line 134, in test_003_checkwav_append_copy
    self.assertEqual(out_params, expected_params)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: _wave[42 chars]e=8000, nframes=8, comptype='NONE', compname='not compressed') != _wave[42 chars]e=8000, nframes=12, comptype='NONE', compname='not compressed')
```


### System Information

Platform: f41
Version: main

### GNU Radio Version

3.11-git (main)

### Specific Version

v3.11.0.0git-889-g0d2f5e434

### Steps to Reproduce the Problem

n/a

### Relevant log output

```shell

```

--- a/gr-blocks/python/blocks/qa_wavfile.py
+++ b/gr-blocks/python/blocks/qa_wavfile.py
@@ -104,6 +104,7 @@ class test_wavefile(gr_unittest.TestCase):
         wf_out.close()
 
         # 3. append halved copy
+        self.tb.disconnect_all()
         wf_in = blocks.wavfile_source(infile)
         halver = blocks.multiply_const_ff(0.5)
         wf_out = blocks.wavfile_sink(outfile,

